deltachat-ios/AppDelegate.swift

```diff
@@ -48,7 +50,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     // `didFinishLaunchingWithOptions` is _not_ called
     // when the app wakes up from "suspended" state
     // (app is in memory in the background but no code is executed, IO stopped)
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication, willFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
moving  `let notificationOption = launchOptions?[.remoteNotification]` to `willFinishLaunchingWithOptions` shouldn't work according to the documentation: https://developer.apple.com/documentation/uikit/uiapplication/launchoptionskey/1622967-remotenotification (see the Discussion part)